src/main.rs

```diff
@@ -13,6 +14,30 @@ pub mod face;
 
 const PORT: u16 = 3333;
 
+
+fn save_image(image: Vec<u8>, name: &str) -> Result<(), io::Error> {
+    let mut pos = 0;
+    let mut image_buffer = match File::create(name) {
+        Ok(file) => file,
+        Err(error) => {
+            error!("file create failed: {}", error);
```

Comment:
Get rid of this line since it will be returned in the error which is being printed from main.